src/features/pilot_management/PilotSheet/sections/mech/components/MechNav.vue

```diff
@@ -3,18 +3,24 @@
     <div id="cap" />
     <div v-if="$vuetify.breakpoint.mdAndUp" class="d-inline">
       <cc-nav-item tile depressed :selected="selected === 0" @click="$emit('set-page', 0)">
-        MECH CONFIGURATION
+        <cc-tooltip inlinde delayed content="Active Mech Configuration">
+          MECH CONFIGURATION
+        </cc-tooltip>
       </cc-nav-item>
       <cc-tooltip simple inline content="Feature In Development">
         <cc-nav-item disabled :selected="selected === 1" @click="$emit('set-page', 1)">
           COMBAT ANALYTICS
         </cc-nav-item>
       </cc-tooltip>
       <v-btn icon fab x-small outlined class="mx-4 unskew" dark @click="toPilotSheet()">
-        <v-icon large>cci-pilot</v-icon>
+        <cc-tooltip inline delayed content="Player Profile">
+          <v-icon large>cci-pilot</v-icon>
+        </cc-tooltip>
```

Comment:
I don't know if we might want to also wrap the whole button in the cc-tooltip for these last ones or not. Does the UX feel any different if you do change the nesting?

Reply:
Yeah I can wrap the tool tip around the v-btn with no ill effects. I'll make that change.